src/test/java/com/intel/gkl/testing_utils/TestingUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.intel.gkl.testing_utils;
```

Comment:
Please use camel format for pkg name

Reply:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
The package names should be snake_cased but I can concatenate it and make `testingutils` instead.